client/src/components/Context.js

```diff
@@ -115,6 +117,15 @@ function getRoutes(user) {
       icon: IconPermIdentity,
       disabled: false,
     },
+
+    {
```

Comment:
Let's move this up to beneath _Verify_, since that's where it sits logically in the workflow.